cmd/ftl/cmd_call.go

```diff
@@ -39,8 +41,43 @@ func (c *callCmd) Run(ctx context.Context, client ftlv1connect.VerbServiceClient
 	if err != nil {
 		return fmt.Errorf("invalid request: %w", err)
 	}
+
+	logger.Debugf("Calling %s", c.Verb)
+	// lookup the verbs
+	res, err := ctlCli.GetVerbs(ctx, connect.NewRequest(&ftlv1.VerbsRequest{}))
+	if err != nil {
+		return err
+	}
+
+	var foundVerb *reflection.Ref
+	suggestions := []string{}
+	verbs := res.Msg.GetVerbs()
+	logger.Debugf("Found %d verbs", len(verbs))
+	needle := []rune(fmt.Sprintf("%s.%s", c.Verb.Module, c.Verb.Name))
+
+	// only consider suggesting verbs that are within 40% of the length of the needle
+	distanceThreshold := int(float64(len(needle))*0.4) + 1
+	for _, verb := range verbs {
+		d := lev.DistanceForStrings([]rune(verb), needle, lev.DefaultOptions)
+		logger.Debugf("Verb %s distance %d", verb, d)
+		// found a match, stop searching
+		if d == 0 {
+			foundVerb = &c.Verb
+			break
+		}
+
+		if d <= distanceThreshold {
+			suggestions = append(suggestions, verb)
+		}
+	}
+
+	// no match found
+	if foundVerb == nil {
+		return fmt.Errorf("verb not found, did you mean one of these: %v", suggestions)
```

Comment:
Change this to the following so users don't get weird brackets in the error:

```suggestion
		return fmt.Errorf("verb not found, did you mean one of these: %s", strings.Join(suggestions, ", "))
```